src/app/script.js

```diff
@@ -0,0 +1,32 @@
+function addTodo(text) {
+  let todoList = document.getElementById('todoList')
+  let entry = document.createElement('li')
+
+  entry.appendChild(document.createTextNode(text))
+  todoList.appendChild(entry)
+}
+
+async function updateTodo() {
+  // update DOM with new list item
+  const textBox = document.getElementById('description')
+
+  const text = textBox.value
+
+  addTodo(text)
+  await postData(text).then(response => console.log(response))
```

Comment:
```suggestion
  await postData(text)
```

On further thought, let's stop printing the response object to the console.  The response object itself is confusing to have in the browser console (because we have to consume the stream manually)